test/functionalTest/cases/1316_create_subscription/post_subscriptions.test

```diff
@@ -186,7 +186,8 @@ Date: REGEX(.*)
             "humidity",
             "temperature"
         ],
-        "callback": "http://localhost:1234"
+        "callback": "http://localhost:1234",
+        "throttling": 5
```

Comment:
There was a bug (now fixed) precluding throttling to appear in the GET subscriptions information?